srcpkgs/hangups/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'hangups'
+pkgname=hangups
+version=0.4.6
+revision=1
+build_style=python3-module
+pycompile_module="hangups"
+hostmakedepends="python3-setuptools"
+depends="python3-MechanicalSoup python3-readlike python3-ReParser python3-Confi
```

Comment:
well cute, at the first space before 80 columns if the 80th column is taking by a non whitespace character.

you just break the dependencies otherwise.

Reply:
so maybe cut at the - or cut at the start of the dependency that breaks 80 columns?
```
depends="python3-MechanicalSoup python3-readlike python3-ReParser
 python3-ConfigArgParse..."
```